docs/posts/flink-unified-stream-batch/index.qmd

```diff
@@ -0,0 +1,414 @@
+---
+title: "Stream-batch unification through Ibis lens"
+author: "Chloe"
+date: 2024-02-08
+categories:
+    - blog
+    - flink
+    - streaming
+---
+
+One of my focuses in the past 10 months has been to implement the Flink backend
+for Ibis. I was working with Apache Flink and building a feature engineering
+tool, and we stumbled upon Ibis as we attempted to build our own translation
+layer that could turn user declarations into relation trees, then optimize and
+deploy the query plan, all while maintaining the underlying infrastructure for
+the user. We considered and prototyped with a number of tools and eventually
+chose Ibis. It had already established a position in the batch world and had
+support for 10+ of the most popular batch engines (at the time). We loved the
+idea of decoupling the user-facing interface from the execution engine, so that
+ users can swap out the execution engine depending on their needs, without
+ having to rewrite code. And, of course, it was open-source. It was everything
+ we dreamed of.
```

Comment:
some form of this could be good for the testimonials page

also unnecessary spaces on lines 20-22